src/ocm/hooks/useCpuArchitectures.ts

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { SupportLevels, useAlerts } from '../../common';
+import { getApiErrorMessage, handleApiError } from '../api';
+import NewFeatureSupportLevelsAPI from '../services/apis/NewFeatureSupportLevelsAPI';
+
+export default function useCpuArchitectures(openshiftVersion?: string) {
+  const [cpuArchitectures, setCpuArchitectures] = React.useState<SupportLevels>();
```

Comment:
I'm confused with the terminology. We're calling this data `cpuArchitectures` but it has type `SupportLevels`. (it happens in a few places, and it's confusing to me)

Reply:
This new object is defined in src/common/api/types:
/**
 * Map of feature ID or CPU architecture alongside their support level
 */
export interface SupportLevels {
  [name: string]: SupportLevel;
}

Reply:
So its type is not CpuArchitecture, why name the variable like that?
If I see a variable named `cpuArchitectures` I would expect it's a list of CPU architectures.